pkgs/development/compilers/jetbrains-jdk/default.nix

```diff
@@ -64,8 +64,8 @@ openjdk17.overrideAttrs (oldAttrs: rec {
   buildPhase = ''
     runHook preBuild
 
-    mkdir -p jcef_linux_${arch}/jmods
-    cp ${jetbrains.jcef}/* jcef_linux_${arch}/jmods
+    mkdir -p jcef_linux_${arch}/
+    cp -r ${jetbrains.jcef}/* jcef_linux_${arch}/
```

Comment:
```suggestion
    cp -r --reflink=auto ${jetbrains.jcef}/* jcef_linux_${arch}/
```

Reply:
Isn't reflink auto the default now? 

>   cp and install now default to copy-on-write (COW) if available.

https://lists.gnu.org/archive/html/coreutils/2021-09/msg00113.html